EnglishFilter/sections/foreign.txt

```diff
@@ -4347,6 +4347,8 @@ dap-news.com##.ads
 !------------------------------ Korean -------------------------------
 !---------------------------------------------------------------------
 !
+! https://github.com/AdguardTeam/AdguardFilters/pull/88347
+bosa.co.kr,econovill.com,sisafocus.co.kr,newsian.co.kr,idsn.co.kr,ggilbo.com,andongdaily.com,tiptipnews.co.kr,fntoday.co.kr,newstown.co.kr,womennews.co.kr,sisajournal.com,zum.com,safetynews.co.kr,koreabiomed.com,mindpost.or.kr,energydaily.co.kr,docdocdoc.co.kr,edaily.co.kr##div.banner_box
```

Comment:
Wry previous rule is bad?
Checked randomly docdocdoc.co.kr - there are no `.banner_box` elements.

Reply:
![scress](https://user-images.githubusercontent.com/37463155/125881018-01b670c2-8ffd-4fdc-b7d7-b498e3054f4b.png)
![wesdf](https://user-images.githubusercontent.com/37463155/125881026-a812c647-5550-4b2f-9908-802a528dc7c5.png)
https://www.docdocdoc.co.kr/news/articleView.html?idxno=2012593
https://www.docdocdoc.co.kr/
No.   There are exist.

Reply:
I think `[...] ##div.banner_box` is more safer.